src/app/gene-plot/gene-plot.component.ts

```diff
@@ -234,6 +247,27 @@ export class GenePlotComponent implements OnChanges {
   }
 
   private drawPlot(): void {
+    //console.log(this.zoomHistory.currentState.yMin);
+    //console.log(this.map(Math.log(this.zoomHistory.currentState.yMin), -8, 4, 0, 280));
+
+    if (this.isInZoom()) {
+      const zoomElement = this.plotElement.append('g').attr('id', 'zoomElement');
+      draw.rect(zoomElement, 0, this.plotWidth,
+        this.map(Math.log(this.zoomHistory.currentState.yMin), -8, 5, 295, -9), 1, '#357ae8', 1, 'rect');
+      draw.rect(zoomElement, 0, this.plotWidth,
+        this.map(Math.log(this.zoomHistory.currentState.yMax), -8, 5, 295, -9), 1, '#1567eb', 1, 'rect');
```

Comment:
magic/unexplained numbers such as ` -8, 5, 295, -9` should be avoided

Reply:
the colors are too similar, just use a single value and extract it to the `constants` object

Reply:
minor nitpick, but change the `rect` labels to something like `ySelectionMin` and `ySelectionMax`